src/components/Menu/Menu.tsx

```diff
@@ -14,6 +15,10 @@ import styles from './Menu.module.css';
 
 // Note: added className here to prevent private interface collision within HeadlessUI
 export type MenuProps = ExtractProps<typeof HeadlessMenu> & {
+  /**
+   * Text to be placed within the Button trigger to open the Menu.
+   */
+  buttonText?: React.ReactNode;
```

Comment:
Not entirely sold on this prop name, could be buttonNode? buttonFill? suggestions?

Reply:
`buttonContent`?

Reply:
No longer going with the `button{prefix}` prop, resolving